components/events/EventsGoodsList.tsx

```
@@ -77,20 +77,22 @@ export const EventsGoodsList = ({ event, user }: { event: Event, user?: TripUser
     return (
         <View style={{ flex: 1 }}>
             <View className="mb-4">
-                <Text className="dark:text-white text-lg">
-                    Liste partagée de l'activité.
-                </Text>
+                <View className="flex-row items-center gap-2">
+                    <IconSymbol name="list.bullet" size={18} color="orange" />
+                    <Text className="dark:text-white text-xl font-bold ">
+                        Liste partagée de l'activité
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)